### ISSUE TITLE
Use JSON for Rockmeter

### ISSUE THREAD

**Issue by erodozer** [fofix/fofix#24]
The current rockmeter implementation I did awhile ago is a complete mess of ini files.  I propose rewriting it to use a simple JSON structure that clearly shows a structured scenegraph when designing rockmeters.  It would also mean one less thing dependent on configparser.

**Comment by mdsitton** [comment]
@nhydock Do you still happen to have that json rockmeter code laying around?

**Comment by nhydock** [comment]
@mdsitton No, I don't think so, but rewriting it is a piece of cake since JSON is clean and my rockmeter code is crap and needs to be redesigned anyway.  It's been awhile since I had a FoFiX build environment set up on my computer.  I'll see what I can do this weekend if I'm not bogged with fixing bugs for my game or adding new features to it.

**Comment by mdsitton** [comment]
Awesome, looking forward to it. be sure to hop in irc if you do get the chance to work on anything.
